src/hierarchies/lapnig_hierarchy.h

```diff
@@ -0,0 +1,125 @@
+#ifndef BAYESMIX_HIERARCHIES_LAPNIG_HIERARCHY_H_
+#define BAYESMIX_HIERARCHIES_LAPNIG_HIERARCHY_H_
+
+#include <google/protobuf/stubs/casts.h>
+
+#include <Eigen/Dense>
+#include <memory>
+#include <set>
+#include <vector>
+
+#include "algorithm_state.pb.h"
+#include "base_hierarchy.h"
+#include "hierarchy_id.pb.h"
+#include "hierarchy_prior.pb.h"
+
+namespace LapNIG {
```

Comment:
This file could use a small introductory text. See other hierarchies such as NNIG for an example

Reply:
We added it, thank you.